eoscostarica/eosio-spend-explorer
# chore(hapi): database persistence of blockchain events

### Database persistence of blockchain events

### What does this PR do?

- Resolve #30 

### Steps to test

1. Run the project
2. Go to the folder hapi/src/services/hyperion/index and see the changes
3. Go to the folder hapi/src/services/hyperion/updaters and see the changes on eosiotoken and genesiseden

#### CheckList
